bin/deseq2.R

```diff
@@ -534,8 +534,8 @@ for (comparison in comparisons) {
   ##########################################
 
   factor <- "condition"
-  numerator <- l2
-  denominator <- l1
+  numerator <- l1
```

Comment:
uhh I know that we switched the order here at some point :) @MarieLataretu is this the correct current state? numerator == l1 and denominator == l2?

Reply:
I think this is actually your change and its just shown here because I rebased this branch after the change on master!

Reply:
@MarieLataretu can you please check what's the order we now agreed on ^^ I remember that you did some change here at some point...

Reply:
Yes, this is fine!